ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/mac_security.schema.yml

```diff
@@ -68,3 +74,19 @@ keys:
                       description: Rekey period in seconds
             sci:
               type: bool
+            l2_protocols:
+              type: list
+              primary_key: name
+              items:
+                type: dict
+                keys:
+                  name:
+                    type: str
+                    valid_values:
+                      - lldp
+                      - ethernet-flow-control
+                  mode:
+                    type: str
+                    valid_values:
```

Comment:
Please add "required: true" under "mode". "name" is implicitly required because it is the primary_key.
This also means that you can remove the test for the variables in the eos cfg template, since they are always set.
Likewise in the documentation template, you can remove the default to '-'

Reply:
Fixed